teku/src/main/java/tech/pegasys/teku/cli/options/RemoteValidatorApiOptions.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.cli.options;
+
+import picocli.CommandLine.Option;
+
+public class RemoteValidatorApiOptions {
+
+  @Option(
+      names = {"--remote-validator-api-enabled"},
```

Comment:
If you prefix these with `--X` they'll turn up when you run `teku -X` so they're easier to find.